3주차/FindAnimal/src/App.jsx

```diff
@@ -1,445 +1,15 @@
-    }
-  };
-  return (
-    <div>
-      <Header>나와 닮은 동물은?</Header>
-      <Group>
-        <TextBox>물놀이 좋아해??</TextBox>
-        <div className="question-box">
-          <Question onClick={() => handleOptionChange("웅!")}> 웅!!</Question>
-          <Question onClick={() => handleOptionChange("반반")}> 반반</Question>
-          <Question onClick={() => handleOptionChange("아니ㅠ")}>
-            {" "}
-            아니ㅠ
-          </Question>
-        </div>
-        <div>
-          <span>
-            <NavigateButton onClick={handleGoPage1}>back</NavigateButton>{" "}
-          </span>
-          <span>
-            <NavigateButton onClick={handleGoPage3} disabled={!buttonActive}>
-              next
-            </NavigateButton>{" "}
-          </span>
-        </div>
-      </Group>
-    </div>
-  );
-}
-
-function Page3({ goPage4, goPage2 }) {
-  const [selectedOption, setSelectedOption] = useState(null);
-  const [buttonActive, setButtonActive] = useState(false);
-
-  const handleOptionChange = (option) => {
-    setSelectedOption(option);
-    setButtonActive(option != null);
-  };
-
-  const handleGoPage2 = () => {
-    goPage2();
-  };
-  const handleGoPage4 = () => {
-    if (selectedOption !== null) {
-      goPage4(selectedOption);
-    } else {
-      alert("동물을 선택해주세요.");
-    }
-  };
-  return (
-    <div>
-      <Header>나와 닮은 동물은?</Header>
-      <Group>
-        <TextBox>귀엽다는 말 들어본 적 있어?</TextBox>
-        <div className="question-box">
-          <Question onClick={() => handleOptionChange("웅!")}> 응!!</Question>
-          <Question onClick={() => handleOptionChange("아니ㅠ")}>
-            {" "}
-            아니.
-          </Question>
-        </div>
-        <div>
-          <span>
-            <NavigateButton onClick={handleGoPage2}>back</NavigateButton>{" "}
-          </span>
-          <span>
-            <NavigateButton onClick={handleGoPage4} disabled={!buttonActive}>
-              결과보기
-            </NavigateButton>{" "}
-          </span>
-        </div>
-      </Group>
-    </div>
-  );
-}
-
-function Page4({ selectedOptions, animalData, restartPage1 }) {
-  function getMatchingAnimal(selectedOptions) {
-    const matchingAnimalData = animalData.find(
-      (data) =>
-        data.answer.length === selectedOptions.length &&
-        data.answer.toString() === selectedOptions.toString()
-    );
-
-    return matchingAnimalData ? matchingAnimalData : "알 수 없음";
-  }
-  const handleRestart = () => {
-    restartPage1();
-  };
-  const matchingAnimal = getMatchingAnimal(selectedOptions).animal;
-  const matchingAnimalPic = getMatchingAnimal(selectedOptions).imgUrl;
-
-  return (
-    <div>
-      <Group>
-        <TextBox>너랑 닮은 동물은.. {matchingAnimal}(이)야!!</TextBox>
-        <AnimalPic src={matchingAnimalPic}></AnimalPic>
-        <div>
-          <ClickButton onClick={handleRestart}>다시하기</ClickButton>
-        </div>
-      </Group>
-    </div>
+    <ThemeProvider theme={theme}>
+      <GlobalStyle />
+      {RenderPage()}
```

Comment:
컴포넌트니까
```suggestion
      <RenderPage/>
```
얘도 JSX 문법으로 이렇게 가져오면 되겠다!